lib/cmd/clean.cc

```diff
@@ -0,0 +1,48 @@
+// std
+#include <iostream>
+#include <string>
+
+// external
+#include <spdlog/spdlog.h> // NOLINT(build/include_order)
+
+// internal
+#include "poac/cmd/clean.hpp"
+#include "poac/config.hpp"
+#include "poac/data/manifest.hpp"
+#include "poac/util/validator.hpp"
+
+namespace poac::cmd::clean {
+
+[[nodiscard]] Result<void>
+clean(const Options& opts) {
+  const auto profile =
```

Comment:
Please use precise types rather than `auto`.

```suggestion
  const Option<String> profile =
```

Reply:
✔️